services/submission.js

```diff
@@ -180,6 +181,33 @@ class Submission {
         }, {"$limit": 1}]);
         return (result?.length > 0) ? result[0] : null;
     }
+
+    async getSubmission(params, context){
+        verifySession(context)
+            .verifyInitialized()
+            .verifyRole([ROLES.SUBMITTER, ROLES.ORG_OWNER, ROLES.DC_POC, ROLES.FEDERAL_LEAD, ROLES.CURATOR, ROLES.ADMIN]);
+        let id = params?._id
+        let rUser = await this.userService.getUserByID(context?.userInfo?._id);
+        const aSubmission = await this.findByID(id);
+        if(context?.userInfo?.role == ROLES.DC_POC){
+            if((rUser?.dataCommons.includes(aSubmission?.dataCommons))){
```

Comment:
instead of nested if statements we can just use a single if statement with an "&&" operator